test/test_utils.dart

```diff
@@ -21,6 +22,10 @@ String getPackagePath() {
 
     _packagePathCache = p.normalize(p.join(p.dirname(currentFilePath), '..'));
   }
+
+  // On Windows, the tests will crash because _packagePathCache starts with a backslash.
+  if (Platform.isWindows && _packagePathCache.startsWith('\\'))
```

Comment:
nit: move this into the if so it doesn't run every time this function is called.